1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -16,8 +17,31 @@ const BIRTHDAYS = [
     "November 15th"
 ];
 
+// COUNTER VARIABLE
+let i = 0;
+
+// HELPER FUNCTION
+function notInJuly (date) {
+    
+    return date.search("July") === -1;
+
+}
+
+// MAIN FUNCTION
 function findFirstJulyBDay(birthdays) {
-    // TODO
+
+    while (i < birthdays.length) {
+
+        if (notInJuly(birthdays[i])) {
+
+            i++;
+            continue;
+
+        }
+
+        return birthdays[i];
+    }
```

Comment:
This will work, but using the 'continue' statement makes the code slightly less readable.

Can you think of another way of implementing this algorithm that doesn't use continue, but uses `if` and the increment statements only instead?

Reply:
Looks good 👍